### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getConfigParam' of undefined

### ISSUE THREAD

**Issue by markmiro** [atom-haskell/ide-haskell-repl#32]
1. Install the package
2. Open a file like this in Atom: "~/haskell/test.hs" (REPL also doesn't open unless I have a .hs file open)
3. Click "Haskell IDE > Open REPL" in the menubar
4. Error occurs.

**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.6
**Thrown From**: [ide-haskell-repl](https://github.com/atom-haskell/ide-haskell-repl) package, v0.1.4
### Stack Trace

Uncaught TypeError: Cannot read property 'getConfigParam' of undefined

```
At /Users/markmiro/.atom/packages/ide-haskell-repl/lib/ide-haskell-repl-view.coffee:62

TypeError: Cannot read property 'getConfigParam' of undefined
  at Immediate._onImmediate (/Users/markmiro/.atom/packages/ide-haskell-repl/lib/ide-haskell-repl-view.coffee:62:11)
  at tryOnImmediate (timers.js:534:15)
  at processImmediate [as _immediateCallback] (timers.js:514:5)

```
### Commands

```
     -0:50.8.0 ide-haskell-repl:toggle (div.package-detail.panels-item)
     -0:29.7.0 settings-view:open (atom-text-editor.editor.is-focused)
  2x -0:04.6.0 ide-haskell-repl:toggle (div.package-detail.panels-item)
```
### Config

``` json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig",
        ".hgrc"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "atom-jade",
      "language-jsx",
      "autocomplete-html",
      "vim-mode",
      "language-javascript-semantic",
      "minimap",
      "tree-view"
    ],
    "themes": [
      "one-dark-ui",
      "2-dark-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
2-dark-syntax, v0.1.1 (active)
atom-jade, v0.3.0 (inactive)
atom-yeoman, v0.3.15 (active)
autocomplete-modules, v1.6.7 (active)
autocomplete-paths, v1.0.2 (active)
base16-ocean-dark-syntax-theme, v0.1.9 (inactive)
color-picker, v2.2.2 (active)
electron-light-syntax, v1.3.0 (inactive)
electron-syntax, v1.2.0 (inactive)
emmet, v2.4.3 (active)
file-icons, v1.7.19 (active)
genesis-ui, v0.5.0 (inactive)
github-web, v1.1.0 (inactive)
gulp-control, v0.5.0 (inactive)
hashrocket, v0.4.10 (active)
ide-haskell-repl, v0.1.4 (active)
language-babel, v2.42.2 (active)
language-haskell, v1.9.9 (active)
language-jade, v0.7.2 (active)
language-javascript-semantic, v0.2.1 (inactive)
linter, v1.11.16 (active)
linter-eslint, v7.3.2 (active)
merge-conflicts, v1.4.4 (active)
minimap, v4.25.0 (inactive)
no-caffeine-syntax, v0.17.2 (inactive)
nuclide, v0.171.0 (active)
octocat-syntax, v0.1.5 (inactive)
php-twig, v4.0.0 (active)
pigments, v0.35.2 (active)
polymorph-ui, v1.8.8 (inactive)
predawn-syntax, v1.0.2 (inactive)
predawn-ui, v1.0.2 (inactive)
react, v0.16.0 (active)
script, v3.10.1 (inactive)
seti-syntax, v1.0.1 (inactive)
seti-ui, v1.3.2 (inactive)
snow-light-syntax, v0.2.0 (inactive)
spacegray-chalk-syntax, v7.0.4 (inactive)
spacegray-dark-syntax, v0.2.0 (inactive)
spacegray-dark-ui, v0.12.0 (inactive)
spaceship-ui, v0.1.0 (inactive)
Sublime-Style-Column-Selection, v1.7.2 (active)
trailing-semicolon, v1.1.0 (inactive)
unity-ui, v2.1.5 (inactive)
vim-mode, v0.65.1 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (active)
one-light-ui, v1.5.0 (inactive)
one-dark-syntax, v1.3.0 (inactive)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
find-and-replace, v0.201.0 (inactive)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.2 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.65.0 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.208.2 (inactive)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by lierdakil** [comment]
Thanks for reporting! Should be fixed in 0.1.5. That said, this package is designed for use alongside ide-haskell & friends (ide-haskell-cabal, etc), so I suggest you consider installing those as well. Standalone operation is supported though, despite an occasional bug, so it's entirely up to you.

**Comment by markmiro** [comment]
Cool! Thanks for the quick response!
